brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/RelationshipModifier.java

```diff
@@ -37,7 +41,10 @@ public void apply(EntitySpec<?> entitySpec, NodeTemplate nodeTemplate, Topology
             RelationshipTemplate relationshipTemplate = findRelationshipRequirement(nodeTemplate, requirementId);
             if (relationshipTemplate != null && relationshipTemplate.getType().equals("brooklyn.relationships.Configure")) {
 
-                Map<String, Object> relationProperties = getTemplatePropertyObjects(relationshipTemplate);
```

Comment:
Since this change is duplicated twice, why not change `getTemplatePropertyObejcts'` arguments from `RelationshipTemplate, PaaSNodeTemplate, Map<String, PaaSNodeTemplate>` to `RelationshipTemplate, TreeBuilder` and do the retrieval of paasNodeTemplate and the Map there?